.github/workflows/java-ec2-asg-test.yml

```diff
@@ -14,6 +14,10 @@ on:
       caller-workflow-name:
         required: true
         type: string
+      cpu-architecture:
+        required: false
+        type: string
+        default: "x86_64"
```

Comment:
nit: We could use the `options` feature to somewhat enforce the input/to let people know what the other option for cpu architecture is

That or just put a comment to let people know

Reply:
I don't see such an option in https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputs

Reply:
Added comment

Reply:
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

It might only be an option for composite actions